ui/client/components/SummaryIntroDialog.js

```diff
@@ -398,7 +400,7 @@ const SummaryIntroDialog = ({
                     type="submit"
                     disableElevation
                     disabled={disableConfirm}
-                    onClick={() => versionBumpModel()}
+                    onClick={(event) => versionBumpModel(event)}
```

Comment:
No need to wrap the function with an anonymous arrow function.
Likely from copy-pasting or habit. Still happens to me occasionally.

Reply:
If the function isn't wrapped with an anonymous arrow function, it gets called when the JSX is calculated. For a button click we do need the wrapper or this happens the second the button shows up on the page. 
https://stackoverflow.com/questions/67529332/how-does-an-anonymous-function-work-in-onclick-in-react-component

Reply:
Oh, no. I meant this:

```
onClick={versionBumpModel}
```

Reply:
Ah, yes, that makes more sense!